### ISSUE TITLE
YOLOX-Backbone Package

### ISSUE THREAD

**Issue by developer0hye** [Megvii-BaseDetection/YOLOX#478]
Thanks for sharing great works!

https://github.com/developer0hye/YOLOX-Backbone

I made the package for the people who want to use the pre-trained backbone of YOLOX.

I inspired by [timm](https://github.com/rwightman/pytorch-image-models).

Would you mind distributing this package?

**Comment by ruinmessi** [comment]
Hi, thx for your excellent work! For LICENSE and README, you may just follow Apache-2.0 to preserve the copyright and license notices.

By the way, we will update the code with ~2x faster training and ~+1% AP for all models within several days. As it contains some backwards incompatible changes (e.g., we remove the normalization operation like -mean/std), I suggest you wait until the next week for these updates and new models.

**Comment by developer0hye** [comment]
@ruinmessi 

Okay! I'm gonna update code after the update you mentioned!

**Comment by developer0hye** [comment]
@ruinmessi 

I updated my package! Thanks for sharing great works!
